lib/truffler.js

```diff
@@ -68,6 +68,9 @@ Truffler.prototype._run = function(url, options, done) {
 		state.timeout = true;
 	}, options.timeout);
 
+	// Log the set options in debug mode
+	options.log.debug('Running with options ' + JSON.stringify(options, null, 4));
```

Comment:
For those like me not familiar with truffler, running what? PhantomJS? Shall we change the text to it's clear?